### DISCUSSION TITLE
litellm models not working in remote docker instance

### DISCUSSION THREAD

**Discussion by jwu-sym** [open-webui/open-webui#2523] [General]
# Bug Report
Litellm seems not be able to proxy stream chat completions in a docker image on a remote host.

## Description
The docker image, litellm works in localhost, stream chat completion.
Same docker image on a remote host, received  internal server error when attempting
```python
INFO:     216.144.248.28:0 - "HEAD / HTTP/1.1" 200 OK
ERROR:apps.litellm.main:400 Client Error: Bad Request for url: http://localhost:14365/v1/chat/completions
Traceback (most recent call last):
  File "/data/source/open-webui/backend/apps/litellm/main.py", line 354, in proxy
    r.raise_for_status()
  File "/data/source/open-webui/backend/venv/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://localhost:14365/v1/chat/completions
 - "POST /litellm/api/v1/chat/completions HTTP/1.1" 500 Internal Server Error
```

it can get 
```
:0 - "GET /litellm/api/v1/models HTTP/1.1" 200 OK , 
```
but not stream request

**Bug Summary:**
litellm models not working in remote docker instance

**Steps to Reproduce:**
deploy the image to a remote host, add a few litellm models with keys, the same configuration works in docker image hosted in local.

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Open WebUI Version:** >[v0.1.118] or after mid April 2024, it has worked before that.
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [ x] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ x] I have included the browser console logs.
- [ x] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
We've had too many issues with bundled LiteLLM support, so It won't be bundled with open-webui anymore for our future release, and you should still be able to add self-hosted LiteLLM instance via Open WebUI OpenAI APIs. Hope that helps!

**Reply by flefevre** [reply]
It will be definitely a good idea not to add litellm directly bundle in Open WebUI.
Do you have a tracker, link on this task?
Any idea on the difficulty to do it?
I will be able to test if necessary.

**Comment by jwu-sym** [comment]
Thank you, good plan.

**Reply by jwu-sym** [reply]
Just wanted to share that for the post-mortem, I resolved my issue by clearing my browser cache, logging out, and then logging back in. It didn't seem to be a problem with the Litellm or Open-WebUI code.

However, it would be great if the logout function could automatically clear the session auth token so we don't have to manually clear our cache or it could be something else. Thanks a bunch!
